inst/qml/GeneralizedLinearModel.qml

```diff
@@ -0,0 +1,469 @@
+                label:		qsTr("Inverse")
+                value:		"inverse"
+                visible:	family.familyMap[family.currentValue].includes(value)
+                checked:	family.familyDefault[family.currentValue] == "inverse"
+            }
+
+            RadioButton
+            {
+                label:		qsTr("1/mu^2")
+                value:		"1/mu^2"
+                visible:	family.familyMap[family.currentValue].includes(value)
+                checked:	family.familyDefault[family.currentValue] == "1/mu^2"
+            }
+        }
+    }
+
+    Section
+    {
+        title: qsTr("Model")
+
+        VariablesForm
+        {
+            preferredHeight: jaspTheme.smallDefaultVariablesFormHeight
+
+            AvailableVariablesList
+            {
+                name: "availableTerms"
+                title: qsTr("Components")
+                source: ['covariates', 'factors']
+                width: parent.width / 4
+            }
+            ModelTermsList { width: parent.width * 5 / 9 }
+        }
+
+        CheckBox { name: "interceptTerm"; label: qsTr("Include intercept"); checked: true }
+    }
+
+    Section
+    {
+        title: qsTr("Statistics")
+
+        Group
+        {
+            title: qsTr("Model Fit")
+            CheckBox {
+                name: "gofDeviance"; label: qsTr("Deviance goodness-of-fit test")
+            }
+            CheckBox {
+                name: "gofPearson"; label: qsTr("Pearson goodness-of-fit test")
+            }
+        }
+
+        Group
+        {
+            title: qsTr("Parameter Estimates")
+            CheckBox { name: "coefEstimates"; label: qsTr("Estimates"); checked: true }
+            CheckBox {
+                name: "coefCi"; label: qsTr("Confidence intervals")
+                childrenOnSameRow: true
+                CIField { name: "coefCiInterval"; }
+            }
+        }
+
+
+    }
+
+    Section
+    {
+        title: qsTr("Diagnostics")
+
+        Group
+        {
+            title: qsTr("Analysis of Residuals")
+            Layout.rowSpan: 3
+            Group
+            {
+                title: qsTr("Deviance Residuals")
+                CheckBox {
+                    name: "devResVsYPlot"; label: qsTr("Residuals vs. fitted")
+                }
+                CheckBox {
+                    name: "devResVsXPlot"; label: qsTr("Residuals vs. predictor")
+                }
+                CheckBox {
+                    name: "devResQqPlot"; label: qsTr("Q-Q plot")
+                }
+            }
+
+            Group
+            {
+                title: qsTr("Pearson Residuals")
+                CheckBox {
+                    name: "prsResVsYPlot"; label: qsTr("Residuals vs. fitted")
+                }
+                CheckBox {
+                    name: "prsResVsXPlot"; label: qsTr("Residuals vs. predictor")
+                }
+                CheckBox {
+                    name: "prsResQqPlot"; label: qsTr("Q-Q plot")
+                }
```

Comment:
This would also benefit from using a custom component to remove the code duplication...